cmd/local.go

```diff
@@ -443,7 +443,16 @@ func executeLocalJob(model *NonMemModel) turnstile.ConcurrentError {
 
 	log.Debugf("Script location is pegged at %s", scriptLocation)
 
-	command := exec.Command(scriptLocation)
+	bash, err := exec.LookPath("bash")
+	if err != nil {
+		return turnstile.ConcurrentError{
+			Error:         err,
+			RunIdentifier: model.FileName,
+			Notes:         fmt.Sprintf("bash is required: %v", err),
```

Comment:
Can we give them a friendlier error here? This is perfectly reasonable for developers, but the most likely people to stumble onto this are scientific users or regulators. I'm not sure exactly what, but something like 

> bash is required to run bbi on Windows. Please install bash and then try running this again.

I was thinking of even throwing in something like "try `which bash` to make sure you have it installed before retrying" but that might be overkill. Especially because... it's not clear where they would even run `which bash` on Windows.

Reply:
I've updated the message to your suggestion, but I left out "on Windows" since it's true for other systems too and possible (even if unlikely to hit this) and, if you're on Windows and seeing this, it seems a bit unnecessary to say.